knowledge-base/chart-draw-horizontal-vertical-lines.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Charts - Draw Horizontal and Vertical Lines
+description: How to draw horizontal lines in Charts. How to draw vertical lines in Charts.
+type: how-to
+page_title: How to Draw Horizontal and Vertical Lines in Charts
+slug: chart-kb-draw-horizontal-and-vertical-lines
+position: 
+tags: telerik, blazor, chart, draw, horizontal, vertical, lines
+ticketid: 1511761, 1569732
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Charts for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to draw horizontal lines in Telerik Blazor Charts?
+* How to draw vertical lines in Telerik Blazor Charts?
+* Is it possible to draw vertical or horizontal lines on the Telerik Blazor Chart?
+
+
+## Solution
+
+There are two ways to draw horizontal and vertical lines on the Telerik Blazor Charts.
+
+### Using Plot Bands
+
+Steps for drawing vertical lines with plot bands:
+
+1. Add `ChartXAxisPlotBands` instance inside the `ChartXAxis`.
+2. Add `ChartXAxisPlotBand` instances inside the `ChartXAxisPlotBands` based on the needed number of lines.
+
+Steps for drawing horizontal lines with plot bands:
+
+1. Add `ChartYAxisPlotBands` instance inside the `ChartYAxis`.
+2. Add `ChartYAxisPlotBand` instances inside the `ChartYAxisPlotBands` based on the needed number of lines.
+
+>caption Drawing Horizontal and Vertical lines with Plot Bands
+
+`````CSHTML
+<TelerikChart>
```

Comment:
Think of ways to add more value to this example, compared to the existing ones in the Plot Bands article. For example, mention if there is a limit on how narrow the plot band can be.